app/static/src/app/components/sensitivity/SensitivityTracesPlot.vue

```diff
@@ -48,6 +51,7 @@ export default defineComponent({
         const store = useStore();
 
         const solutions = computed(() => store.state.sensitivity.result?.batch?.solutions || []);
+        const graphConfig = computed(() => store.state.graphs.config[0]);
```

Comment:
Same question - unclear to me why we know that the relevant graph config is the first one.
We should do something that ensures that the `0` of line 13 and the `0` of this line are always equal: e.g. move this `computed` into the child so that the graphIndex prop is used to look up the graphConfig, or set a graphIndex variable in this component that both line 13 and this line can reference.

Reply:
Again, we'll sort this in mrc-5485, it can just look at the first config until then.